process_vanguard_bibs.py

```diff
@@ -89,6 +89,7 @@ def move_9xx(record):
         new_field = field_mapping[old_field]
         for fld in record.get_fields(old_field):
             fld.tag = new_field
+            record.remove_field(fld)
```

Comment:
@aprigge I think the `get_5xx_fields()` function also needs updating to remove the old field.

Reply:
The `get_5xx_fields()` just adds the key pairs to the dictionary so the `record.remove_field(fld)` in `change_CLU()` removes the old field

Reply:
Ah, yes, I read that wrong.  Thanks for fixing the right thing!